lib/perlfaq4.pod

```diff
@@ -1116,6 +1116,32 @@ signal that I missed something:
 
     print $string;
 
+
+=head2 Does Perl have anything like Ruby's #{} or Python's f string?
+
+Unlike the others, Perl allows you to embed a variable naked in a quoted string C<"variable $variable">, when there isn't whitespace around the variable name you can add braces like C<"foo ${foo}bar">.
+
+A list can also be embedded directly in a string, and will be expanded by default with spaces between the elements. The default expansion can be changed by assigning a different string to the special variable B<$">, such as C<$" = ', ';>.
+
+Perl also supports references within a string providing the equivalent of the features in the other two languages.
+
+C<${\ ... }> embedded within a string will work for most simple statements such as an object->method call. More complex code can be wrapped in a do block C<${\ do{...} }>. 
```

Comment:
I think the author test is picking up a trailing space character here.

Reply:
In my checkout of the repo the test 'xt/release/changes_has_content.t' is missing.

Reply:
Grinnz was referring to xt/author/eol.t. It's a generated test' you can run it via `dzil test`, or if you don't want to install Dist::Zilla and its many prereqs, you can get it from the latest cpan release of perlfaq.

Reply:
I just copied the tests from the cpan distribution, cpanm can't install CreateQuestionList when I try to install the authordeps.

Another test was failing because perlfaq.pod is missing, I copied that back from the current release on cpan to get it to pass.

all of the tests are now passing locally for me with prove and yath but not with dzil test. xt/release/changes_has_content is failing with dzil

 dzil authordeps | cpanm
! Finding inc::CreateQuestionList on cpanmetadb failed.
! Finding inc::CreateQuestionList () on mirror http://www.cpan.org failed.
! Couldn't find module or a distribution inc::CreateQuestionList

Reply:
`inc::CreateQuestionList` is locally bundled so it's ok that it failed to install. authordeps can't tell the difference.